support/approved-access.adoc

```diff
@@ -0,0 +1,31 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="approved-access"]
+= Approved Access
+include::_attributes/common-attributes.adoc[]
+ifdef::openshift-dedicated[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+endif::[]
+:context: approved-access
+
+toc::[]
+
+Red{nbsp}Hat Site Reliability Engineering (SRE) typically does not require access to systems containing customer data as part of normal operations to manage and support {product-title} clusters. In the unlikely event that SRE needs access to systems containing customer data, you can use the _Approved Access_ interface to review and _approve_ or _deny_ access to these systems.
```

Comment:
I'm not blocking merge on this, but overall per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference), we can use underscores/italics for a "first reference" / "first term" style appearance of something (in this case, this first appearance of the feature name _Approved Access_), but after that it should just be Approved Access (no italics).

That is to say, I don't think considering it "general emphasis" is appropriate for subsequent instances.

<img width="775" alt="image" src="https://github.com/openshift/openshift-docs/assets/3442316/93794a43-08b6-4d64-a975-0f166e3686a2">

However, if it ever shows up as an actual web console / UI element or page name or similar, then it should be asterisks/bold (which you have appropriately done elsewhere).